otx/algorithms/classification/adapters/mmcls/models/backbone/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""Classification backbones."""
```

Comment:
Have we deicded to move 'backbones' to `common/adapters/mmcv/configs/bacbkones`? please cross-check @harimkang 

ASAIK, For the detection, segmentation, all backbones are implemented in `common/`

Reply:
We didn't decide and I was heard it does not matter as now. I don't know if it's right to move it to mmcv. These models are modules, not yaml files, registered in mmcls. I heard the mmdet lastest version can be compatible with mmcls models, not sure if works in current version. I hope Harim handle this (move or version up) for his next task.

Reply:
These files seem to confuse reviewers. In my recommendation, The backbone is moved by adding `mmcls of the common adapter (or mmcv)`, and it seems good to remove the head and classifier first. But I'm not sure. I think you should proceed as you are sure.

Reply:
The reason I recommend `common` is that detection can also use this backbone. Also, since `mmcls` is a required installation library of `mpa`, I think it can belong to `common`.

Reply:
Moved backbones to common/adapters/mmcls/models. Could you please check again?